### PR TITLE
Owners for developer tools

### PR DESCRIPTION
**Pull request by maximedenes**
This PR adds owners for scripts in `dev/tools`. We need secondary maintainers for most of them, please volunteer!

### CODE REVIEW

**Comment by maximedenes** [comment]
Maybe we can merge this without waiting for secondary maintainers volunteers, since it is already better than the current status.

**Comment by herbelin** [comment]
Looks like the shell script experts are already there indeed. Tentative suggestion: arbitrarily putting the names of @skyskimmer, @gares and @jasongross to fill the holes in a compatible way (???), or @Zimmi48.

**Review by Zimmi48 — APPROVED** [review]
I'll merge later today if no more secondary maintainers can be found.

**Comment by maximedenes** [comment]
Wait, I made a wrong push to this branch.

**Comment by maximedenes** [comment]
Now fixed.

**Pull request merged**
